provider/pkg/gen/schema.go

```diff
@@ -718,6 +723,19 @@ func (g *packageGenerator) genProperties(typeName string, typeSchema *discovery.
 	return &result, nil
 }
 
+func (g *packageGenerator) isCopyFromOutput(propName string) bool {
```

Comment:
Nit - rename to `shouldCopyFromOutput`?  Current name suggests something was copied already.